jbpm-benchmarks/kieserver-performance-tests/src/main/java/org/jbpm/test/performance/taskassigning/updates/TaskAssigningUpdates.java

```diff
@@ -31,6 +31,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
+import com.codahale.metrics.Meter;
```

Comment:
I looks like this import is not being used

Reply:
thanks